### PR TITLE
Optional param include_diag for bivariate feature functions

### PR DESCRIPTION
**Pull request by jbschiratti**
Added `include_diag=False` parameter in bivariate feature functions

Addresses #11.

### CODE REVIEW

File: mne_features/univariate.py [hunk 1]
```diff
@@ -29,7 +31,10 @@ def get_univariate_funcs(sfreq):
     Returns
     -------
     univariate_funcs : dict
+        The dictionary of feature functions.
     """
+    # XXX : you should make this automatically by inspecting the available
+    # functions that start with compute_
```
Comment (agramfort) [inline mne_features/univariate.py]: for this @jbschiratti you could use something along these lines:

```
def get_univariate_funcs(sfreq):
    univariate_funcs = dict()
    for var in dir(univariate):
        if var.startswith("compute_"):
            var_short = var[len("compute_"):]
            univariate_funcs[var_short] = getattr(univariate, var)

    return univariate_funcs
```

that I would put in the `mne_features/__init__.py` file

Comment (jbschiratti) [reply mne_features/univariate.py]: I totally agree! Now that the aliases of feature functions are clearly defined, `univariate_funcs` and `bivariate_funcs` dicts should be created automatically.

File: mne_features/utils.py [hunk 2]
```diff
@@ -158,3 +165,37 @@ def filt(sfreq, data, filter_freqs, verbose=False):
...
+    res = getmembers(sys.modules[module_name], isfunction)
+    for name, func in res:
+        if 'compute_' in name:
+            alias = name.split('compute_')[-1]
+            if hasattr(func, 'func_code'):
+                func_code = func.func_code
+            else:
+                func_code = func.__code__
+            args, _, _ = getargs(func_code)
+            if args[0] == 'sfreq':
```
Comment (agramfort) [inline mne_features/utils.py]: sfreq may not be first. use `'sfreq' in args`

File: mne_features/utils.py [hunk 3]
```diff
@@ -158,3 +165,37 @@ def filt(sfreq, data, filter_freqs, verbose=False):
...
+        Name of the module to inspect.
+
+    Returns
+    -------
+    feature_funcs : dict
+    """
+    feature_funcs = dict()
+    res = getmembers(sys.modules[module_name], isfunction)
+    for name, func in res:
+        if 'compute_' in name:
```
Comment (agramfort) [inline mne_features/utils.py]: `if name.startswith('compute_')`

File: mne_features/utils.py [hunk 4]
```diff
@@ -158,3 +165,37 @@ def filt(sfreq, data, filter_freqs, verbose=False):
         return filter_data(data, sfreq=sfreq, l_freq=filter_freqs[0],
                            h_freq=filter_freqs[1], picks=None,
                            fir_design='firwin', verbose=_verbose)
+
+
+def _get_feature_funcs(sfreq, module_name):
+    """ Inspects a given module and returns a dictionary of feature
```
Comment (agramfort) [inline mne_features/utils.py]: pep257

doc should be first 1 short line. THen an empty line and a short paragraph.

**Pull request merged**